core/src/no/group15/playmagic/PlayMagic.kt

```diff
@@ -10,16 +10,31 @@ import no.group15.playmagic.ui.AppState
 import no.group15.playmagic.ui.views.MainMenuView
 
 
+const val WINDOW_WIDTH = 1280
+const val WINDOW_HEIGHT = 720
+
 class PlayMagic(private val logLevel: Int) : ApplicationListener {
 
 	private lateinit var appState: AppState
 	private lateinit var injectContext: Context
 
+	// Application wide input commands
 	private val commonInput = object : InputAdapter() {
 		override fun keyUp(keycode: Int): Boolean {
-			if (keycode == Input.Keys.ESCAPE || keycode == Input.Keys.BACK) {
+			// Uses the back button on Android or escape key to navigate "back" with the state defining what back means
+			if (keycode in setOf(Input.Keys.ESCAPE, keycode == Input.Keys.BACK)) {
```

Comment:
Shouldn't it just be `setOf(Input.Keys.ESCAPE, Input.Keys.BACK)`?

Reply:
oh, bugger, fixed